src/guides/v2.3/install-gde/prereq/prereq-overview.md

```diff
@@ -83,7 +83,7 @@ For example:
 mysql -u magento -p
 ```
 
-You must run MySQL version 5.6 or later as the following result indicates:
+You must run MySQL 5.7 for on-premise installations. Magento is also compatible with, but has not been tested and is not recommended, with MySQL 5.6, MariaDB 10.1, and Percona 5.7. The following result indicates which version you are running:
```

Comment:
I would query the statement that Magento has not been tested on MariaDB 10.1 - Magento Cloud runs on MariaDB 10.1 / 10.2 : https://devdocs.magento.com/cloud/project/services-mysql.html

I feel that this needs to be justified.

Reply:
I think it's best defer to the main system reqs page instead of repeating versions in different places; it makes maintenance really difficult. The main system reqs page is the source of truth about what versions of third-party software Magento Commerce and Open Source have been tested with. These may not be the same as Magento Cloud.

Also, we have a sentence at the top of the 2.4 main system requirements page that directs users with cloud projects to a different service version resource. Perhaps we need a similar note on the 2.3 system requirements page.

> If you are working on a Magento Commerce Cloud project, see [Service versions](https://devdocs.magento.com/cloud/project/services.html#service-versions) in the Cloud Guide.

https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements-tech.html

Reply:
Perhaps the line should be -
Check you have the correct version of mySQL for the version of Magento you are installing, [Check Here For Supported Versions](https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements-tech.html#database). The following result indicates the version you are running.

This can also be updated on the 2.4 Guide

Reply:
@DominicWatts Great! Can you make the same change to the 2.4 file please? It's not symlinked